notebook/01_tutorial.ipynb

```diff
@@ -97,127 +122,82 @@
+    "    'inputs': ['left_merge_1']\n",
     "}\n",
     "\n",
     "task_outputCsv2 = {\n",
-    "    'id': 'node_outputCsv2',\n",
+    "    'id': 'output_csv_2',\n",
     "    'type': 'OutCsvNode',\n",
     "    'conf': {\n",
     "        'path': 'symbol_returns.csv'\n",
     "    },\n",
-    "    'inputs': ['node_leftMerge2']\n",
+    "    'inputs': ['left_merge_2']\n",
     "}"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "A task schema defined in python is a dictionary with the following fields: `id`, `type`, `conf`, `inputs`, and `filepath`. Additionally there is a `load` and a `save` field used for running workflows (described later on). The `id` for a given task must be unique within a workflow. Tasks use the `id` field in their `inputs` field for specifying that the output(s) of other tasks are inputs. The `type` is the class of the compute task or plugin. The gquant framework already implements a number of such plugins. These can be found in `gquant.plugin_nodes`. For example:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 3,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "class VolumeFilterNode(Node):\n",
-      "\n",
-      "    def columns_setup(self):\n",
-      "        self.required = {\"asset\": \"int64\",\n",
-      "                         \"volume\": \"float64\"}\n",
-      "        self.addition = {\"mean_volume\": \"float64\"}\n",
-      "\n",
-      "    def process(self, inputs):\n",
-      "        \"\"\"\n",
-      "        filter the dataframe based on the min and max values of the average\n",
-      "        volume for each fo the assets.\n",
-      "\n",
-      "        Arguments\n",
-      "        -------\n",
-      "         inputs: list\n",
-      "            list of input dataframes.\n",
-      "        Returns\n",
-      "        -------\n",
-      "        dataframe\n",
-      "        \"\"\"\n",
-      "\n",
-      "        input_df = inputs[0]\n",
-      "        volume_df = input_df[['volume', \"asset\"]].groupby(\n",
-      "                [\"asset\"]).mean().reset_index()\n",
-      "        volume_df.columns = [\"asset\", 'mean_volume']\n",
-      "        merged = input_df.merge(volume_df, on=\"asset\", how='left')\n",
-      "        if 'min' in self.conf:\n",
-      "            minVolume = self.conf['min']\n",
-      "            merged = merged.query('mean_volume >= %f' % (minVolume))\n",
-      "        if 'max' in self.conf:\n",
-      "            maxVolume = self.conf['max']\n",
-      "            merged = merged.query('mean_volume <= %f' % (maxVolume))\n",
-      "        return merged\n",
-      "\n"
-     ]
-    }
-   ],
-   "source": [
-    "import inspect\n",
-    "from gquant.plugin_nodes.transform import VolumeFilterNode\n",
+    "In Python, gQuant nodes are defined as a dictionary with the following fields:\n",
```

Comment:
"In Python, gQuant nodes are defined as a dictionary with the following fields:"
It's a task-spec not a node. Should be:
"In Python, gQuant a task-spec is defined as a dictionary with the following fields:"
But they should use the "TaskSpecSchema" for these fields instead of strings.

Instead of "workflow" it should be "task-graph".

Nodes are plugins or operators. Nodes are instantiated in task-graph when building or running a task-graph.